packages/gluestick-cli/new.js

```diff
@@ -59,16 +60,24 @@ module.exports = (appName, options, exitWithError) => {
   );
 
   spawn.sync(
-    options.yarn ? 'yarn' : 'npm',
+    !options.npm && which('yarn') !== null ? 'yarn' : 'npm',
```

Comment:
Maybe if the user does not pas `--npm` flag and has no `yarn`, we could print a warning like? `Installing dependencies using npm, consider installing yarn` or something like that.